nucypher/policy/conditions/evm.py

```diff
@@ -140,32 +148,49 @@ def chain(self) -> str:
 
     def validate_method(self, method):
         if method not in self.ALLOWED_METHODS:
-            raise Exception(f'{method} is not a permitted RPC endpoint for conditions.')
+            raise ReencryptionCondition.InvalidCondition(
+                f"{method} is not a permitted RPC endpoint for conditions."
```

Comment:
```suggestion
                f"{method} is not a permitted RPC endpoint for condition evaluation."
```

Reply:
If we don't know what other methods to include in the allowed list right now (besides `balanceOf` and eth_getBalance`), let's track it with an issue and corresponding TODO.

Reply:
Should we also provide a list of currently allow-listed methods in the exception message?

Reply:
Might get a bit long over time depending on how https://github.com/nucypher/tdec/issues/64 shakes out. However, we can revisit then.